styles.css

```diff
@@ -1,3 +1,3 @@
 h1 {
-    color: red;
-}
\ No newline at end of file
+  color: red;
```

Comment:
[OPTIONAL] It is considered bad practice to use color names. Consider changing it to its `hex` or `rgb` equivalent